### PR TITLE
cli and operation doc for licence

### PR DESCRIPTION
**Pull request by h12w**

### CODE REVIEW

**Comment by actionbuddha** [comment]
I think the way to go about this is:

- Obtaining a Development License via the GUI -> a few screenshots of the developer journey (get account, generate license)
- Obtaining a Development License via the CLI -> much as you have it (no GUI, just the cli stuff you have written)

That flows quite logically, either you use the GUI or the CLI.

All this is for getting a Development license; we should also put in a small section at the end covering Enterprise Licensing:

'Customers can purchase StorageOS by capacity or node-based pricing. Contact info@storageos.com to discuss.'

**Comment by h12w** [comment]
Refactors done.

File: _docs/operations/licencing.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+```bash
+$ echo PASTE-THE-LICENCE-KEY-HERE | storageos licence apply --stdin
+```
+
+Read the [licence CLI command reference]({% link _docs/reference/cli/licence.md
+%}) for further information.
+
+## Obtaining an Enterprise licence
+
+Please contact [info@storageos.com](mailto:info@storageos.com) to discuss
```
[the remark this answers is not in the document]
Comment (actionbuddha) [reply _docs/operations/licencing.md]: Thanks - confirmed with sales, sales@ is preferred.

File: _docs/operations/licencing.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+---
+layout: guide
+title: StorageOS Docs - Licencing
+anchor: operations
+module: operations/licencing
```
[the remark this answers is not in the document]
Comment (actionbuddha) [reply _docs/operations/licencing.md]: Changed everywhere - I think we're consistent now...

**Comment by actionbuddha** [comment]
@croomes would you mind taking another peek at this when you have time?

File: _guides/005-operations.md [hunk 3]
```diff
@@ -24,6 +24,10 @@ sections:
     title: Rules
     description: StorageOS rules
     module: operations/rules
+  -
+    title: Licencing
```
Comment (croomes) [inline _guides/005-operations.md]: typo

File: _docs/operations/licensing.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+This CLI command can print the cluster ID:
+
+```bash
+storageos licence inspect | jq -r .[].clusterID
+```
+
+To obtain a licence for your StorageOS cluster, create a new StorageOS account
+or log into [StorageOS Portal](https://my.storageos.com), go to the [Licences
+page](https://my.storageos.com/licenses) and follow the instructions on the
```
Comment (croomes) [inline _docs/operations/licensing.md]: unfortunate url...  should be /licences

Comment (actionbuddha) [reply _docs/operations/licensing.md]: Indeed - I caught that, but changing it might be out of scope for now...

**Pull request merged**